libraries/bsg_manycore.h

```diff
@@ -475,15 +475,6 @@ extern "C" {
          */
         int hb_mc_manycore_disable_dram(hb_mc_manycore_t *mc);
 
-
-        /**
-         * Check if NPA is in DRAM.
-         * @param[in]  mc     A manycore instance initialized with hb_mc_manycore_init()
-         * @param[in]  npa    A valid hb_mc_npa_t
-         * @return One if the NPA maps to DRAM - Zero otherwise.
-         */
-        int hb_mc_manycore_npa_is_dram(hb_mc_manycore_t *mc, const hb_mc_npa_t *npa);
-
```

Comment:
Not sure where to put this. I moved it to the DMA API, but it might be better in the NPA header.

Reply:
What was it removed from this header?

Reply:
I wasn't sure where it should be. I thought it was used in the DMA implementation, it was not. I moved it into the bsg_manycore file instead of the header because I wasn't sure the users should be exposed to it. 

Do users use this method?

I can add it back here

Reply:
Did you check the regressions?

Reply:
Yeah, nada.

Reply:
ok - well I guess this is fine then

Reply:
Do you want me to move it back?

Reply:
No.